src/lib/UI/Config/Provider/User.php

```diff
@@ -46,7 +47,7 @@ public function getConfig(): array
         $config = ['user' => null, 'profile_picture_field' => null];
         $token = $this->tokenStorage->getToken();
 
-        if ($token instanceof UsernamePasswordToken) {
+        if ($token instanceof UsernamePasswordToken || $token instanceof PostAuthenticationGuardToken) {
```

Comment:
I agree. @webhdx @Nattfarinn  what is your take on this one?

Reply:
It won't work with `TokenInterface` or `AbstractToken` that is why I did not do it. You will get a `AnonymousToken` which is also a `TokenInferface` and it will fail with:
`Call to a member function getAPIUser() on string` because with a `AnonymousToken` getUser return a string.
There plenty of other ways to fix that issue but not with `TokenInferface`  ;-)

My PR is a real quick fix, valid to me as Guard is becoming a real simple way to implement third parties login/autlogin.

A better way could be to check the Class of `getUser`. It is prob the best way actually. I can do it, let me know if I or you update the PR!

Reply:
As @Plopix said it won't work with the interface and this is the reason why we didn't do it that way. Maybe checking `getUser` against `null` is a good solution but I'd need to investigate if this is valid in all cases. I'll get back to you later @Plopix.